### PR TITLE
Update Non-EVM-Blockchains.md

### PR DESCRIPTION
**Pull request by captainhook0626**
Ethereum doesn't use PoW anymore.

### CODE REVIEW

File: Non-EVM-Blockchains.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Now, we will look at a few different Layer 1 blockchains and try to compare them
 
 ![](https://i.imgur.com/QBhOJRz.png)
 
-NEAR is a Layer-1 smart contract platform that utilizes parallel sharding storage and computation technology to scale. It also uses Proof of Stake consensus, instead of Ethereum's current Proof of Work, and has over 50 validators securing millions of dollars on the network.
+NEAR is a Layer-1 smart contract platform that utilizes parallel sharding storage and computation technology to scale. It also uses Proof of Stake consensus and has over 50 validators securing millions of dollars on the network.
```
Comment (yanukadeneth99) [inline Non-EVM-Blockchains.md]: Hi, thanks for the change, could you change the validator count from 50 to 100 as well? [Proof](https://near.org/about/network/decentralize/)

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]

File: Non-EVM-Blockchains.md (hunk 1 above)
Comment (captainhook0626) [reply Non-EVM-Blockchains.md]: Done, I fixed a few other typos too. Please validate.

File: Lottery-Game-Chainlink-VRF.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ The official Chainlink Docs describe VRFs as:
 - After max number of players have entered the game, one winner is chosen at random
 - The winner will get `maxplayers*entryfee` amount of ether for winning the game
 
-## BUIDL IT
+## BUILD IT
```
Comment (yanukadeneth99) [inline Lottery-Game-Chainlink-VRF.md]: [Buidl](https://academy.binance.com/en/glossary/buidl) is an actual word in web3.

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]

File: Lottery-Game-Chainlink-VRF.md (hunk 2 above)
Comment (captainhook0626) [reply Lottery-Game-Chainlink-VRF.md]: reverted, thank you.

**Comment by yanukadeneth99** [comment]
Looks good, nice work @captainhook0626

**Pull request merged**